packages/thirdweb/src/pay/buyWithFiat/getQuote.ts

```diff
@@ -91,6 +91,14 @@ export type GetBuyWithFiatQuoteParams = {
    * By default, we choose a recommended provider based on the location of the user, KYC status, and currency.
    */
   preferredProvider?: FiatProvider;
+
+    /**
+   * Optional parameter to specify an overriding onramp provider.
+   *
+   * By default, we choose a recommended provider based on the location of the user, KYC status, and currency.
+   * If this is set, we will always choose this specific onramp provider, or error if this provider cannot support the user.
+   */
+  providerOverride?: FiatProvider;
```

Comment:
its weird to have both options. i think if ppl expect to be able to override, then lets just change the behavior of preferred to error if its not available